### PR TITLE
Testing rspec acceptance test fix

### PR DESCRIPTION
**Pull request by zonArt**
Again, not intended to be merged, just to test the acceptance test (sorry for the pollution but if you have better idea to test toggle your CI, I'd be happy to know, I tried making the PR against forked master but it doesn't trigger travis)

### CODE REVIEW

**Comment by zonArt** [comment]
It's really strange that the build is green on master as this "test pr" doesn't change anything in fact and travis fails, what's happening with the beaker tests ? Is it possible that it lacks isolation in some way ? I'm actually trying to make the opened PR #583 green as it doesn't look like the failing test is linked to my commits (was green before I rebase master on it and solved conflicts)

**Comment by jhoblitt** [comment]
There have been lots of issues with the beaker tests under travis due to what I suspect are timing issues.  However, the failure mode on the last commit is new-to-me.  I've restarted the travis run to see if it is reproducible.

File: spec/acceptance/xtypes/jenkins_credentials_spec.rb [hunk 1]
```diff
@@ -52,6 +52,7 @@
           # Run it twice and test for idempotency
           apply(pp, :catch_failures => true)
           apply(pp, :catch_failures => true)
+          #apply(pp, :catch_failures => true)
```
Comment (rtyler) [inline spec/acceptance/xtypes/jenkins_credentials_spec.rb]: o_O

**Comment by jhoblitt** [comment]
Looks like the beakage is due to gem version changes.  I'll see if I can resolve it.

File: spec/acceptance/xtypes/jenkins_credentials_spec.rb (hunk 1 above)
Comment (zonArt) [inline spec/acceptance/xtypes/jenkins_credentials_spec.rb]: It's actually just to test that master is working. In fact, if you check my first commit, I initially tried just to comment the second apply as that was the error I got within PR #583 (failed on line 54 in the beaker test). And now, the test even don't start

**Comment by zonArt** [comment]
@jhoblitt: thanks, just tell me if I can help in some way

**Comment by jhoblitt** [comment]
It looks like `beaker-puppet_install_helper` had a release yesterday that changed it's deps to allow beaker 3.  We may need to pin the beaker version.

**Comment by jhoblitt** [comment]
This is odd but bundler was installing `beaker-rspec` 2.0.0 for some reason while 5.6.0 is the current version.  5.6.0 requires beaker 2, and this combo seems to be working.

**Comment by jhoblitt** [comment]
https://github.com/jenkinsci/puppet-jenkins/pull/644 may be a fix.

**Comment by zonArt** [comment]
@jhoblitt you're right #644 is actually the fix to make the tests work on #583, therefore closing this unneeded PR

**Pull request closed without merging**